osbenchmark/resources/default-operations.json.j2

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "index-append",
+    "operation-type": "bulk",
+    "bulk-size": {{bulk_size | default(5000)}},
+    "ingest-percentage": {{ingest_percentage | default(100)}}
+},
+{
+    "name": "default",
+    "operation-type": "search",
+    "index": {{ indices | map(attribute='name') | list | join(',') | tojson }},
+    "body": {
+        "query": {
+            "match_all": {}
```

Comment:
It might be good to add a `size` parameter to `match_all` queries with a default of 10, which should not alter behavior, but permits the user to run the query with larger response payloads.  This is not specific to this workload, we should change all the existing ones in this fashion.